src/content/docs/manuals/datahub/datahub-arc-sharing.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Sharing ARCs via the DataHUB
+lastUpdated: 2024-03-06
+authors:
+  - dominik-brilhaus
+---
+
+## About this guide
+
+In this guide we explain the different ways offered by the DataHUB to share an ARC with collaborators.
+
+## Before we can start
+
+:ballot_box_with_check: You have a [DataPLANT](https://register.nfdi4plants.org) account
+
+## DataHUB concepts and terminology
+
+Before jumping into the details of data sharing, we need to introduce a few DataHUB concepts, settings and terms that tend to be confusing for new users.
+
+### "Projects" and "Groups" are not the same
+
+**Project = ARC**
+
+- In the DataHUB, ARCs are called "projects"; they are the same.
+- An ARC can be shared with individual users (invited as "members") or a group.
+
+**DataHUB Groups**
+
+- A "Group" is a group of users (then "members" of that group) with specific [permissions](#roles-and-permissions).
+- A group (e.g. your lab or consortium) can share ARCs.
+- Groups help you communicate with all group members and manage permissions for the ARCs shared in that group.
+- Depending on their assigned roles, members of the group can view, edit, delete, and / or maintain the ARCs of that group.
+- If someone has access to a group, they get access to all the ARC in the group.
+- The group Owner(s) can adjust group settings and manage group memberships.
+- Groups can have subgroups (with a subset of members). Subgroups have the same features as groups.  
+
+### Namespaces
+
+- In the DataHUB, namespaces help organize related projects.
+- Every user has a personal namespace, where they can upload or create new ARCs.
+- Every [group and subgroup](#datahub-groups) has an own namespace, respectively.
+
+You can check the namespace by looking at the URL. For example:
+
+<div class="table-container">
```

Comment:
this will not work anymore as we do not use bulma

Reply:
but i think the logic comes out of the box now, so you might be good by just removing this